tests/unit/CMakeLists.txt

```diff
@@ -151,6 +151,10 @@ target_include_directories(${test_prefix}query_procedure_py_module PRIVATE ${CMA
 add_unit_test(query_procedures_mgp_graph.cpp)
 target_link_libraries(${test_prefix}query_procedures_mgp_graph mg-query storage_test_utils)
 target_include_directories(${test_prefix}query_procedures_mgp_graph PRIVATE ${CMAKE_SOURCE_DIR}/include)
+
+add_unit_test(mgp_log.cpp)
+target_link_libraries(${test_prefix}mgp_log mg-query)
+#target_include_directories(${test_prefix}mgp_log PRIVATE ${CMAKE_SOURCE_DIR}/include)
```

Comment:
As the function is in MGP API, I think this unit tests is not good. Check out `tests/e2e/write_procedures` how to write e2e tests for procedures.